Nodejs/Product/Nodejs/TestFrameworks/mocha/mocha.js

```diff
@@ -53,33 +58,74 @@ var run_tests = function (testName, testFile, workingFolder, projectFolder) {
         return;
     }
 
-    var mocha = new Mocha();
-    mocha.ui('tdd');
-    //set timeout to 10 minutes, because the default of 2 sec might be too short (TODO: make it configurable)
-    mocha.suite.timeout(600000);
+    var mocha = initializeMocha(Mocha, projectFolder);
+
     if (testName) {
         mocha.grep(testName);
     }
     mocha.addFile(testFile);
 
-    // Choose 'tap' rather than 'min' or 'xunit'. The reason is that
-    // 'min' produces undisplayable text to stdout and stderr under piped/redirect, 
-    // and 'xunit' does not print the stack trace from the test.
-    mocha.reporter('tap');
-
     mocha.run(function (code) {
         process.exit(code);
     });
 };
 
+function logError(errorMessage, error) {
```

Comment:
Looking at this again... it looks this function is just trying to emulate console.error, but prepend an argument. So a more flexible way to do this would be something like:
``` javascript
var errorArgs = Array.prototype.slice.call(arguments).unshift("NTVS_ERROR:");
console.error.apply(errorArgs);
```